mock-api/db.backup.json

```diff
@@ -0,0 +1,63 @@
+{
+  "users": [
```

Comment:
db.backup not updated

Database user table contains
id
firstName
lastName
passwordHash
email
isEnabled
role (user, admin)

Reply:
Added